helpers/auth.py

```diff
@@ -200,6 +200,17 @@ def login(self):
         if self.config.auth_username:
             username_element = self.fill_username()
 
+        if self.config.auth_form_type == 'MULTI_STEP':
+            log(f"Waiting for {self.config.auth_password_field_name} element to load")
```

Comment:
think it's good to keep these logs in place, it will make the logs more clear if a customer has an issue